### PR TITLE
Display Warning if TechDocs Annotation is Missing

### PR DESCRIPTION
**Pull request by alandovskis**
## Hey, I just made a Pull Request!

Screen Shot 2020-09-07 at 2 22 47 PM

#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [x] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

**Comment by stefanalund** [comment]
Thanks! We should merge this now but later on align with the standardised “empty state” we are working on in https://github.com/spotify/backstage/issues/2179

**Review by stefanalund — APPROVED** [review]

**Review by shmidt-i — APPROVED** [review]

**Review by OrkoHunter — COMMENTED** [review]
Thanks @alandovskis ! I am wondering if we can test this new feature. Would you be able to write one?
When #2179 is there, we can modify both the test and the card.

**Comment by alandovskis** [comment]
I will brush up on TypeScript and Jest and then try to write a test.

**Comment by emmaindal** [comment]
Nice!! 😄 🎉  I wonder if we could add a link to the banner that points to the [getting started guide](https://backstage.io/docs/features/techdocs/creating-and-publishing).

**Review by emmaindal — COMMENTED** [review]
Nice!! 😄 🎉  I wonder if we could add a link to the banner that points to the [getting started guide](https://backstage.io/docs/features/techdocs/creating-and-publishing).

**Comment by alandovskis** [comment]
Screen Shot 2020-09-08 at 9 57 38 PM

**Review by stefanalund — APPROVED** [review]
Thanks and welcome to Backstage! 🎉

File: plugins/techdocs/src/Router.tsx [hunk 1]
```diff
@@ -37,6 +40,20 @@ export const Router = () => {
...
 export const EmbeddedDocsRouter = ({ entity }: { entity: Entity }) => {
+  const projectId = entity.metadata.annotations?.[TECHDOCS_ANNOTATION];
+
+  if (!projectId) {
+    return (
+      <WarningPanel title="Techdocs plugin:">
+        <pre>{TECHDOCS_ANNOTATION}</pre> annotation is missing on the entity.
+        <br />
+        <a href="https://backstage.io/docs/features/techdocs/creating-and-publishing">
+          Getting Started
```
Comment (emmaindal) [inline plugins/techdocs/src/Router.tsx:51]: Nice 🥳 Thank you for your contribution!!

**Review by emmaindal — APPROVED** [review]

**Pull request merged**